### DISCUSSION TITLE
Undefined reference to `read_image_data`

### DISCUSSION THREAD

**Discussion by Hayrpey** [Estia-1a/projetGenieInfo_public#79] [Project Installation and Configuration] [answered]
Bonjour je n'arrive pas à compiler lorsque j'appuie sur build. Le problème vient de 'read_image_data'.
Si vous pouvez m'aider ?

Merci
![image](https://user-images.githubusercontent.com/98600505/173851610-c3628ac3-46de-4d43-9b53-a5168d113fe1.png)

**Comment by WilliamDelamare** [comment]
Usually a problem with the libs version (64 instead of 32).
Try to simply download, unzip and replace your libs with the other version.

**Comment by Hayrpey** [comment]
Super c'était ça merci beaucoup

**Comment by dhmmasson** [comment, marked as answer]
Usually a problem with the libs version (64 instead of 32) : 
1. Check what kind of compiler you are using in the blue ribbon at the bottom of VSCode : On the image above [GCC 4.6.1 i686-pc-**mingw32**], 
2. Download the 32bit version [in the list](https://github.com/Estia-1a/projetGenieInfo_public/releases/tag/v2.0.1)
3. Replace estia-image folder with the correct version (note rename the folder ```estia-image```, it should contain directly inside a folder ```lib``` and a folder ```include```) 
4. Delete the ```build``` folder
5. Build the project using cmake inside vscode (see your readme to do so)
